platform/example.anchor-config.yaml

```diff
@@ -54,6 +54,12 @@ app-config:
     enabled: true
     customerIntegrationEndpoint: http://localhost:8081
 
+  # sep-24
+  sep24:
+    enabled: true
+    interactiveJwtExpiration: 3600
+    interactiveUrl: http://localhost:8081/sep24/interactive
+
```

Comment:
❗  Plz don't forget to add this to:
* default-config.yaml
* anchor-platform-server.yaml
* stellar/kube for previews
* stellar/kube for dev deploy.

Reply:
Thanks for the reminder. I need to make this my muscle memory.

Where is `anchor-platform-server.yaml`.

Reply:
Oh nevermind. ~`anchor-platform-server.yaml`~ doesn't need to be updated.

Reply:
Oh, actually the one place it's missing is [`test-anchor-config.yaml`](https://github.com/stellar/java-stellar-anchor-sdk/blob/ea78c056bef6bbffcf07678075d5d5bc96482d80/integration-tests/src/test/resources/test-anchor-config.yaml)

Reply:
This is the part that I wish to fix in the future. We have too many similar YAMLs floating around.